docs/modelsearch.rst

```diff
@@ -412,6 +414,41 @@ MFL support the following model features:
 +---------------+-------------------------------+--------------------------------------------------------------------+
 | LAGTIME       | None                          | Absorption lagtime                                                 |
 +---------------+-------------------------------+--------------------------------------------------------------------+
+| COVARIATE     | `parameter`, `covariate`,     | Covariate effects                                                  |
+|               | `effect`                      |                                                                    |
++---------------+-------------------------------+--------------------------------------------------------------------+
+
+
+.. _mfl_symbols:
+
+Symbols
+~~~~~~~
+
+The MFL supports defining symbols via the syntax `LET(SYMBOL, [...])`.  For
+instance, to declare a list of absorption parameters use `LET(ABSORPTION, KA)`.
+Those symbols can then be referred to when declaring covariate effects via the
+syntax `COVARIATE(@SYMBOL, ...)`. Certain symbols have an automatic definition
+when not manually defined:
+
++-----------------+-------------+----------------+---------------------------------------------+
+| Alias           | Type        | Definition     | Description of automatic definition         |
++=================+=============+================+=============================================+
+| `@IIV`          | Parameter   | auto or manual | All parameters with a corresponding IIV ETA |
++-----------------+-------------+----------------+---------------------------------------------+
+| `@ABSORPTION`   | Parameter   | auto or manual | All PK absorption parameters                |
++-----------------+-------------+----------------+---------------------------------------------+
+| `@ELIMINATION`  | Parameter   | auto or manual | All PK elimination parameters               |
++-----------------+-------------+----------------+---------------------------------------------+
+| `@DISTRIBUTION` | Parameter   | auto or manual | All PK distribution parameters              |
++-----------------+-------------+----------------+---------------------------------------------+
+| `@CONTINUOUS`   | Covariate   | auto or manual | All continuous covariates                   |
++-----------------+-------------+----------------+---------------------------------------------+
+| `@CATEGORICAL`  | Covariate   | auto or manual | All categorical covariates                  |
++-----------------+-------------+----------------+---------------------------------------------+
```

Comment:
Since all of the aliases can be either automatically or manually defined, would it be clearer without the Definition column?

Reply:
Yep. And I'll rename Alias to Symbol in the header.